tasks/add-assets-repo.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (gulp, config) => () => {
```

Comment:
AFAICT, this script is never used?

Reply:
+1 it is used to execute "git remote add" - it will be used by automated builds as we need to add assets repo to fetch a particular assets branch from it. Devs would likely invoke this only once